hpx/runtime/actions/detail/action_factory.hpp

```diff
@@ -17,26 +17,26 @@
 #include <unordered_map>
 #include <vector>
 
-namespace hpx { namespace actions { namespace detail
-{
+namespace hpx { namespace actions { namespace detail {
+
     struct action_registry
```

Comment:
This shouldn't be needed when networking is disabled.

Reply:
Right... I wasn't sure if this is still needed or not.